recipes/libtlx/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "libtlx" %}
+{% set version = "0.5.20191212" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "5e67d3042a390dbb831b6d46437e3c7fadf738bff362aa7376b210b10ecd532d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - url: 'https://github.com/tlx/tlx/archive/v{{version}}.tar.gz'
+    {{hash_type}}: {{hash_value}}
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake  >=3.5
+    - make
+
+test:
+  commands:
+    # Verify headers are present
+    - test -f $PREFIX/include/tlx/algorithm.hpp  # [unix]
+
+    # Verify pkg-config file is present
+    - test -f $PREFIX/lib/pkgconfig/tlx.pc  # [linux32]
+    - test -f $PREFIX/lib64/pkgconfig/tlx.pc  # [linux64]
```

conda-forge only supports 64 bit os's and puts everything in `lib/`
```suggestion
    - test -f $PREFIX/lib/pkgconfig/tlx.pc  # [linux]
```

---

Thanks for the comments (learning a lot!). 

That makes sense - the initial commit had the line like you suggested, however the build-test failed because of installing in lib64 (related to the proposed change in `build.sh`).